src/sentry/testutils/factories.py

```diff
@@ -834,3 +838,34 @@ def create_platform_external_issue(group=None, service_type=None,
             display_name=display_name,
             web_url=web_url,
         )
+
+    @staticmethod
+    def create_incident(
+        organization, projects, identifier=None, detection_uuid=None, status=0,
+        title=None, query='test query', date_started=None, date_detected=None,
+        date_closed=None,
+    ):
+        # TODO: Remove this once we handle creating this as part of the object
+        if identifier is None:
+            identifier = Incident.objects.filter(organization=organization).count() + 1
```

Comment:
Could you run into concurrency races here?

Reply:
This is just tests, so should be fine? I'm going to write this properly later on when we handle actually creating the incidents.